### PR TITLE
fix typo in examples/instance_segmentation/eval_sbd.py

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: examples/instance_segmentation/eval_sbd.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        '--model', choices=('fcis_resnet101'),
+        '--model', choices=['fcis_resnet101'],
         default='fcis_resnet101')
-    parser.add_argument('--pretrained_model')
```
Comment (yuyu2172) [inline examples/instance_segmentation/eval_sbd.py]: This was correct because it is consistent with other examples.

Comment (knorth55) [reply examples/instance_segmentation/eval_sbd.py]: how about changing all others?

File: examples/instance_segmentation/eval_sbd.py [hunk 2]
```diff
@@ -14,9 +14,9 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        '--model', choices=('fcis_resnet101'),
+        '--model', choices=['fcis_resnet101'],
```
Comment (Hakuyume) [inline examples/instance_segmentation/eval_sbd.py]: For the consistency with others, `('fcis_resnet101',)`. Actually, both list and tuple are acceptable. I prefer using tuple when mutability is not required.

File: examples/instance_segmentation/eval_sbd.py (hunk 1 above)
Comment (Hakuyume) [reply examples/instance_segmentation/eval_sbd.py]: I agree with @knorth55. `--foo-bar` style is more common.

Comment (yuyu2172) [reply examples/instance_segmentation/eval_sbd.py]: OK

File: examples/instance_segmentation/eval_sbd.py (hunk 2 above)
Comment (knorth55) [reply examples/instance_segmentation/eval_sbd.py]: I used to use `tuple` but it shows weird output `  --model {f,c,i,s,_,r,e,s,n,e,t,1,0,1}` as below;
```bash
$ python eval_sbd.py --help
usage: eval_sbd.py [-h] [--model {f,c,i,s,_,r,e,s,n,e,t,1,0,1}]
                   [--pretrained_model PRETRAINED_MODEL]
                   [--iou-thresh IOU_THRESH] [--gpu GPU]

optional arguments:
  -h, --help            show this help message and exit
  --model {f,c,i,s,_,r,e,s,n,e,t,1,0,1}
  --pretrained_model PRETRAINED_MODEL
  --iou-thresh IOU_THRESH
  --gpu GPU
```
When I use `list`, it does not occur.

File: examples/instance_segmentation/eval_sbd.py (hunk 1 above)
Comment (knorth55) [reply examples/instance_segmentation/eval_sbd.py]: move to #616

File: examples/instance_segmentation/eval_sbd.py (hunk 2 above)
Comment (Hakuyume) [reply examples/instance_segmentation/eval_sbd.py]: Perhaps, you forgot `,`.
OK: `('fcis_resnet101',)` (this is a tuple)
NG: `('fcis_resnet101')` (this is a string)

Comment (knorth55) [reply examples/instance_segmentation/eval_sbd.py]: I see, I will modify to use `tuple`

**Pull request merged**